providers/nitro/nitro-helper/src/enclave_log_server.rs

```diff
@@ -95,23 +73,13 @@ impl LogServer {
 
     fn process_log(&mut self, raw_log: &[u8]) -> Result<(), String> {
         let log = Log::from_raw(raw_log).map_err(|e| format!("{:?}", e))?;
-        let mut s = log.format();
-        if self.to_console {
-            match log.level {
-                Level::TRACE => trace!("{}", s),
-                Level::DEBUG => debug!("{}", s),
-                Level::INFO => info!("{}", s),
-                Level::WARN => warn!("{}", s),
-                Level::ERROR => error!("{}", s),
-            }
-        }
-
-        if let Some(file) = self.log_file.as_mut() {
-            let now = Local::now();
-            s = format!("{} {:<6}{}\n", now.format("%F %T%.3f"), log.level, s);
-            file.write_all(s.as_bytes())
-                .map_err(|e| format!("write log to file error: {:?}", e))?;
-            let _ = file.flush();
+        let s = log.format();
+        match log.level {
```

Comment:
This'll change the format of logs?

Reply:
partially... the original log record is what's written by https://github.com/crypto-com/tmkms-light/blob/main/providers/nitro/nitro-helper/src/tracing_layer.rs from inside the enclave.... I guess the tracing macros will add the host's code context